hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestAlterTable.scala

```diff
@@ -169,4 +168,77 @@ class TestAlterTable extends HoodieSparkSqlTestBase {
       }
     }
   }
+
+  test("Test Alter Rename Table") {
+    withTempDir { tmp =>
+      Seq("cow", "mor").foreach { tableType =>
```

Comment:
yes, you are right, I can remove here. Considering the mor table will need other check, for example should we need check _ro and _rt table path, but I think it's better fix in other issue like fix rename mor table, it will be add to check here.

Reply:
> yes, you are right, I can remove here. Considering the mor table will need other check, for example should we need check _ro and _rt table path, but I think it's better fix in other issue like fix rename mor table, it will be add to check here.

I think 🤔 both will better